relayer/crates/starknet-integration-tests/src/tests/clear_packets.rs

```diff
@@ -1,1013 +0,0 @@
-            balance_starknet_b_step_0.quantity + (transfer_quantity * 2).into()
-                - (transfer_back_quantity * 2).into(),
-            balance_starknet_b_step_2.quantity
-        );
-
-        // Assert all packets have been cleared after auto relaying
-        let cosmos_latest_height = cosmos_chain.query_chain_height().await?;
-        let starknet_latest_height = starknet_chain.query_chain_height().await?;
-
-        let (cosmos_commitment1, _) =
-            <CosmosChain as CanQueryPacketCommitment<StarknetChain>>::query_packet_commitment(
-                cosmos_chain,
-                &cosmos_channel_id,
-                &IbcPortId::transfer(),
-                &Sequence::from(1),
-                &cosmos_latest_height,
-            )
-            .await?;
-
-        let (cosmos_commitment2, _) =
-            <CosmosChain as CanQueryPacketCommitment<StarknetChain>>::query_packet_commitment(
-                cosmos_chain,
-                &cosmos_channel_id,
-                &IbcPortId::transfer(),
-                &Sequence::from(2),
-                &cosmos_latest_height,
-            )
-            .await?;
-
-        let (cosmos_commitment3, _) =
-            <CosmosChain as CanQueryPacketCommitment<StarknetChain>>::query_packet_commitment(
-                cosmos_chain,
-                &cosmos_channel_id,
-                &IbcPortId::transfer(),
-                &Sequence::from(3),
-                &cosmos_latest_height,
-            )
-            .await?;
-
-        let (starknet_commitment1, _) =
-            <StarknetChain as CanQueryPacketCommitment<CosmosChain>>::query_packet_commitment(
-                starknet_chain,
-                &starknet_channel_id,
-                &IbcPortId::transfer(),
-                &Sequence::from(1),
-                &starknet_latest_height,
-            )
-            .await?;
-
-        let (starknet_commitment2, _) =
-            <StarknetChain as CanQueryPacketCommitment<CosmosChain>>::query_packet_commitment(
-                starknet_chain,
-                &starknet_channel_id,
-                &IbcPortId::transfer(),
-                &Sequence::from(2),
-                &starknet_latest_height,
-            )
-            .await?;
-
-        assert!(
-            cosmos_commitment1.is_none(),
-            "cosmos_commitment1 expected to be None"
-        );
-        assert!(
-            cosmos_commitment2.is_none(),
-            "cosmos_commitment2 expected to be None"
-        );
-        assert!(
-            cosmos_commitment3.is_none(),
-            "cosmos_commitment3 expected to be None"
-        );
-        assert!(
-            starknet_commitment1.is_none(),
-            "starknet_commitment1 expected to be None"
-        );
-        assert!(
-            starknet_commitment2.is_none(),
-            "starknet_commitment2 expected to be None"
-        );
-
-        Ok(())
-    })
-}
-
-#[test]
-fn test_relay_timeout_packet() -> Result<(), Error> {
```

Comment:
Could we keep this test until it’s migrated to the new format?

Reply:
Added them back now.